pop-api/integration-tests/src/fungibles/mod.rs

```diff
@@ -256,7 +256,7 @@ fn decrease_allowance_works() {
 		// Token does not exist.
 		assert_eq!(
 			decrease_allowance(&addr, 0, &BOB, amount),
-			Err(Module { index: 52, error: [3, 0] }),
+			Err(Module { index: 52, error: [10, 0] }),
```

Comment:
The comment above this assert is outdated if you simply correct the error code. 

Can you please check your own PRs before you ask for a final review. I have been correcting this same mistake in your other PR as well.